draft-altanai-tsv-interop_network_and_application_layer_tunnels.md

```diff
@@ -46,106 +46,106 @@ Tunneling encapsulating and encrypting the original protocol with its own protoc
 
-Some instances of lower layer protocol 
+Some instances of lower layer protocol
 
-* Generic Routing Encapsulation (GRE) [RFC2784]
-* IP-in-IP[RFC1853] 
+* Generic Routing Encapsulation (GRE) {{?RFC2784}}
+* IP-in-IP {{?RFC1853}}
 * IPSec
 * WireGuard
-* OpenVPN 
+* OpenVPN
 * AutoVPN
 
-Some instances of upper layer protocols 
+Some instances of upper layer protocols
 
-* SSH and  Secure Real-time Transport Protocol (SRTP) [RFC 5764] operate at the application layer. 
+* SSH and  Secure Real-time Transport Protocol (SRTP) {{?RFC5764}} operate at the application layer.
 * Stream Control Transmission Protocol (SCTP)
 * DTLS.
 
-- SSL/TLS tunnelling 
- 
-- A L3 tunneling protocol, such as IPSec [RFCs 1825–1829, RFCs 2401–2412], can provde secure path to almost all protocols in the TCP/IP suite. IPSec is a frame work of open standards to provide secure and private IP networks. Authentication Header (AH) and Encapsulating Security Payload (ESP) are part of the IPSec protocol suite, where Security Parameter Index(SPI) uniquely identifies a connetion and each packet in the flow bears a Sequence Number (SN) and a integtrity checksum called ICV (integrity check value). ESP packets inside UDP packets traversing through NATs[RFC3948]
+- SSL/TLS tunnelling
+
+- A L3 tunneling protocol, such as IPSec ({{?RFC4301}}), can provde secure path to almost all protocols in the TCP/IP suite. IPSec is a frame work of open standards to provide secure and private IP networks. Authentication Header (AH) and Encapsulating Security Payload (ESP) are part of the IPSec protocol suite, where Security Parameter Index(SPI) uniquely identifies a connetion and each packet in the flow bears a Sequence Number (SN) and a integtrity checksum called ICV (integrity check value). ESP packets inside UDP packets traversing through NATs {{?RFC3948}}.
```

Comment:
I picked RFC 4301 by seeing what obsoleted 1825, but a quick search suggests that RFC 6071 might be a better informative reference here. Regardless, it's not generally possible to put a range of RFCs in a citation.

Reply:
I see  Noted . Thank you for pointing that out